nservicebus/nhibernate/custom-mappings.md

```diff
@@ -0,0 +1,153 @@
+---
+title: NHibernate Custom mapping with hbm.xml, attributes or fluent api
+summary: Create a custom mapping to change your NHibernate creates the database schema using different techniques.
+tags:
+ - NHibernate
+ - Persistence
+ - Configuration
+ - Mapping
+related:
+ - nservicebus/nhibernate/accessing-data
+ - nservicebus/nhibernate/index
+---
+
+
+Sometimes the default schema that gets generated for your saga does work. For example, you might need to store strings with a different string length, want so use complex types or want to store DateTime values with a high precision or maybe want to tweak the eager or lazy loading to improve performance. You will need to create a custom mapping to achieve this.
```

Comment:
> Sometimes the default schema that gets generated for your saga does work. For example, you might n

You are missing some text here? it read like you are giving examples of when the default schema is correct when you are actually giving examples of the opposite